All project code/index.js

```diff
@@ -112,25 +112,32 @@ app.get("/profile", (req, res) => {
     let month;
     let day;
     let year;
+    let week;
+    let currDay;
+    let today = new Date();
+    currDay = today.getDate();
 
-    if(req.query.month && req.query.day && req.query.year) {
+    if(req.query.month && req.query.day && req.query.year && req.query.week) {
         month = parseInt(req.query.month);
         day = parseInt(req.query.day);
         year = parseInt(req.query.year);
+        week = parseInt(req.query.week);
     }
     else {
-        let today = new Date();
         today.setDate(today.getDate() - today.getDay());
         month = today.getMonth() + 1;
```

Comment:
I was reading the code and found this unintuitive. Mind adding this comment?

```suggestion
        month = today.getMonth() + 1; // getMonth counts 0-11, where January=0
```